configs/train_config.yaml

```diff
@@ -0,0 +1,53 @@
+#task: contrastive_lecun_classes
+task: ae_contrastive_lecun_classes
+# options: ae, contrastive_lecun_classes, ae_contrastive_lecun_classes, ae_triplet_classes, ae_triplet_hierarchical_classes
+
+# Model name (e.g. IncrFeatStridedConvFCUpsampReflectPadAE)
+model_name: IncrFeatStridedConvFCUpsampReflectPadAE
+
+trk_path:
+hdf5_path: "/media/fdumais/My_Passport/Acuity/Data/TractoinfernoExtended/HDF5_QBx/data_tractoinferno_qbx_reduced.hdf5"
+pickle_path:
+rbx_classes: False
+
+# Reference anatomical filename (usually a t1.nii.gz or wm.nii.gz)
+ref_anat_fname: "/home/fdumais/Desktop/Data/finta_rbx_atlas/mni_masked.nii.gz"
+
+# Dataset name (e.g. `fibercup`, `ismrm2015_phantom`, etc.)
+dataset_name: mni_brain
```

Comment:
We may need to rethink this. By `dataset_name` I meant a dataset that is listed in some `anatomy/*.json` and `data/lut/*.json` to automatically e.g. color bundles with a prefixed colormap or obtain long names for datasets. `mni_brain` is not really indicative of any dataset out there; maybe you used it as a default/unimportant name.

Reply:
To do in another PR